perllib/FixMyStreet/Cobrand/TfL.pm

```diff
@@ -530,7 +530,23 @@ sub _cleaning_categories { [
     'Fly tipping - Enforcement Request',
     'Graffiti and Flyposting', # Bromley
     'Street cleaning/sweeping', # Camden
-
+    # Southwark have a variety of flytipping and a their own Street Litter
+    'Street Litter Picking',
```

Comment:
I think we might want the entire list of Southwark street cleaning categories (all those with service codes `STCL_*`) as they cover more than just flytipping. This list also needs to be the category's full name, and not just the display name (Southwark has a lot of identically display-named categories but they actually have a `(Street)` or `(Estates)` suffix in their name to differentiate them).

Full list from the db is:

```
Abandoned Bike (Street)
Clear Broken Glass (Street)
Clinical Waste - Dead Animal (Street)
Dog Waste Bin - Emptying
Dog Waste Clearance (Street)
Dog Waste - Signs
Fly Posting - General (Street)
Fly Posting - Racist/Obscene (Street)
Flytipping - Animal Carcass (Street)
Flytipping - Asbestos (Street)
Flytipping - Black Bags (Street)
Flytipping - Blk Bags Commercial (Street)
Flytipping - Builders Rubble (Street)
Flytipping - Chemical Drums (Street)
Flytipping - Clinical (Street)
Flytipping - Drug related Litter (Street)
Flytipping - General (Street)
Flytipping - Green Waste (Street)
Flytipping - Other Electrical (Street)
Flytipping - Other (Unidentified) (Street)
Flytipping - Tyres (Street)
Flytipping - Vehicle Parts (Street)
Flytipping - White Goods (Street)
Graffiti - General (Street)
Graffiti - Racist / Obscene
Litter Bin - Emptying
Litter Bin - Move or Remove
Other (Street)
Street - Furniture Cleaning
Street - Leafing Request
Street Litter Picking
Street - Weeding Request
Tree Issues on the Highway
```